tv_apps/smart-system/js/preview_window.js

```diff
@@ -147,6 +147,7 @@
 
   PreviewWindow.prototype._handle__willdestroy = function(evt) {
     this.container.element.removeEventListener('_closed', this);
+    this.element.removeEventListener('_loaded', this);
```

Comment:
Did you try to load another page within the iframe? I imagine that `_loaded` should be triggered again and the hint message will show twice incorrectly.